### DISCUSSION TITLE
Put -arch in wsx file?

### DISCUSSION THREAD

**Discussion by StevenABrown** [wixtoolset/issues#6563] [Questions]
Is there a line of code I can put in the wsx file that will invoke the "-arch x64" switch to candle.exe, so that I don't have to type "-arch x64" on the command line?

**Comment by robmen** [comment]
Not really. Why do you want to do that?

**Reply by StevenABrown** [reply]
I installed the WiX extension to Video Studio and found a setting in VS that adds the -arch x64 switch to the command line of candle.exe.
